src/Connector.SqlServer/Utils/SqlName.cs

```diff
@@ -17,15 +17,15 @@ public static SqlName FromUnsafe(string value)
             if (string.IsNullOrEmpty(value)) throw new ArgumentException("Value cannot be null or empty.", nameof(value));
 
             var sanitizedValue = value.ToSanitizedSqlName();
-            if (string.IsNullOrEmpty(sanitizedValue))
+            if (string.IsNullOrEmpty(sanitizedValue)) 
             {
                 throw new ArgumentException("Name cannot be empty after being sanitized", nameof(value));
             }
 
             return new SqlName(sanitizedValue);
         }
 
-        public static SqlName FromSanitized(string value)
+        public static SqlName FromSanitized(string value) 
```

Comment:
@zvirja from other POV, this method looks like backdoor. Consumer of API could create SqlName with any value (and corrupted one) if he wants or not paying enough attention.